code/game/objects/items/stacks/cash.dm

```diff
@@ -12,6 +12,19 @@
 	w_class = 1
 	burn_state = FLAMMABLE
 	var/value = 1
+	var/cooldown = 0
+
+/obj/item/stack/spacecash/attack_self(mob/user as mob)
+	if(!canconsume(M, user))
+		return 0
```

Comment:
What is this check supposed to do? Also, what's this `M` and where does it come from? (Copy-pasting code without checking it afterwards is bad).

`canconsume(mob/eater, mob/user)` is defined for items of type `/obj/item/weapon/reagent_containers/` as per https://github.com/tgstation/tgstation/blob/master/code/modules/reagents/reagent_containers.dm#L55